src/seedu/addressbook/data/person/UniquePersonList.java

```diff
@@ -110,6 +110,15 @@ public void add(Person toAdd) throws DuplicatePersonException {
         internalList.add(toAdd);
     }
 
+    /**
+     * Sorts all persons in the list.
+     *
+     */
+    public UniquePersonList sort() {
```

Comment:
this method should not return "this" for better encapsulation. Instead, it should return a boolean value indicating whether the sorting was successful or not